### ISSUE TITLE
`Create new chat from template` does not work everywhere

### ISSUE THREAD

**Issue by lukemt** [bramses/chatgpt-md#62]
The `Create new chat from template` command does not show up when the welcome page or canvas is open. It only seems to appear inside of markdown files.

Steps to reproduce:
- Open a new Tab
- Welcome page appears
- Try to invoke the command

**Comment by bramses** [comment]
I agree, it is caused by `editorCallback` which exists in arbitrary places in the code. It may be a small change, but I'd have to test it doesn't break anything downstream. Thank you for the PR btw!!! :)

**Comment by lukemt** [comment]
Sure thing! Yes do that, I already tested it though
